src/NativeScript/Runtime/ReleasePool.h

```diff
@@ -13,7 +13,7 @@
 #include <typeindex>
 #include <vector>
 #include <map>
-#include <stack>
+#include <wtf/Deque.h>
```

Comment:
For the love of me I couldn't make it to work with an `std::stack`, even with a liberal sprinkling of `std::move`. We can team up for a `std::stack` based implementation, but I'm not sure the effort is worth it.